assets/airports/eglc.json

```diff
@@ -76,52 +942,67 @@
     "DVR": {
       "icao": "DVR",
       "name": "Dover Five",
-      "suffix": {"09":"", "27":""},
+      "suffix": {"09":"5U", "27":"5T"},
       "rwy": {
-        "09": ["GINTI", "DET", "DVR"],
-        "27": ["GINTI", "DET", "DVR"]
+        "09": [["LCE01", "A570+|S210"], ["LCE02", "A3000|S210"], "BEMID"],
```

Comment:
Follow-up to inline comments on #413: In the unusual situation like this where there actually exists a restriction of 570 feet (usually I haven't seen precision exceed 100ft increments except very rarely), you can't express it with full precision in flight levels (either FL005 or FL006). But the way the logic is implemented, you can preserve the precision by simply entering `"A5.7+|S210"` instead of `"A570+|S210"`. It will multiply out properly.